barretenberg/acir_tests/flows/sol.sh

```diff
@@ -1,14 +1,18 @@
 #!/bin/sh
 set -eu
 
+VFLAG=${VERBOSE:+-v}
+BFLAG="-b ./target/program.json"
+FLAGS="-c $CRS_PATH $VFLAG"
+
 export PROOF="$(pwd)/proof"
 export PROOF_AS_FIELDS="$(pwd)/proof_fields.json"
 
 # Create a proof, write the solidity contract, write the proof as fields in order to extract the public inputs
-$BIN prove -o proof
-$BIN write_vk  -o vk
-$BIN proof_as_fields -k vk -c $CRS_PATH -p $PROOF
-$BIN contract -k vk -c $CRS_PATH -b ./target/program.json -o Key.sol
+$BIN prove -o proof $FLAGS
+$BIN write_vk  -o vk $FLAGS
+$BIN proof_as_fields -k vk $FLAGS -p $PROOF
+$BIN contract -k vk $FLAGS $FLAGS -o Key.sol
```

Comment:
is one of these supposed to be BFLAG?

Reply:
yes it indeed was, thanks for catching! The default is ./target/program.json so things didn't go wrong.